portal-api/unit.jest.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
   collectCoverageFrom: ['src/**/*.{js,}'],
   coverageDirectory: 'generated_reports/coverage/unit',
-  testMatch: ['**/*.test.js'],
+  testMatch: ['**/login.controller.test.js'],
```

Comment:
```suggestion
  testMatch: ['**/*.test.js'],
```
I think this was a mistake :)